### PR TITLE
update-create-user-mailbox-task4455945

### PR DESCRIPTION
**Pull request by Lovina-Saldanha**
Removed section "How do you know this worked?"

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/recipients-in-exchange-online/create-user-mailboxes.md [hunk 1]
```diff
@@ -53,21 +53,3 @@ New-Mailbox -Alias hollyh -Name hollyh -FirstName Holly -LastName Holt -DisplayN
 ```
 
 After you create a mailbox by running the previous command, an user account is also created. You have to activate this user account by assigning a license. To assign a license in the Microsoft 365 admin center, see [Add users individually or in bulk](https://docs.microsoft.com/microsoft-365/admin/add-users/add-users).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/create-user-mailboxes.md:55]:
```suggestion
After you create a mailbox by running the previous command, a user account is also created. You have to activate this user account by assigning a license. To assign a license in the Microsoft 365 admin center, see [Add users individually or in bulk](https://docs.microsoft.com/microsoft-365/admin/add-users/add-users).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - although not related to this PR, please also correct the typo in the preceding paragraph as indicated.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**